docs/pyproject.toml

```diff
@@ -5,9 +5,9 @@ dependencies = [
     "furo==2023.3.27",
     "gitpython==3.1.31",
     "myst-parser==1.0.0",
-    "Sphinx==5.3.0",  # Can't upgrade to Sphinx 6 because sphinx-copybutton 0.5.1 requires Sphinx < 6.
```

Comment:
Is this no longer a problem? I thought I ran into the same issue when I tried to build docs last week

Reply:
It seems the copybutton limitation was removed some time ago. But, this change should be in a separate PR.